src/nginxconfig/i18n/zh-cn/templates/app.js

```diff
@@ -29,6 +29,9 @@ import common from '../common';
 export default {
     title: `${common.nginx} 配置`,
     description: `配置高性能、安全、稳定的${common.nginx}服务器的最简单方法。`,
+    en: '英语', // TODO: translate
+    zhCN: '简体中文', // TODO: translate
+    zhTW: '繁体中文', // TODO: translate
```

Comment:
👋 @moniang I was wondering if you could check these translations are correct (or suggest new ones)?

The source strings are:
```
    en: 'English',
    zhCN: 'Chinese (simplified)',
    zhTW: 'Chinese (traditional)',
```

Reply:
Yes, no problem